### ISSUE TITLE
jetty-util is needed for building

### ISSUE THREAD

**Issue by hawkwang** [pulsarIO/jetstream#12]
It seems that the pom.xml of jetstreamcore need jetty-util.
So it is better to add dependency as follows:
---------------------------------------------------------------
		<dependency>
			<groupId>org.mortbay.jetty</groupId>
			<artifactId>jetty-util</artifactId>
			<version>6.1.26</version>
		</dependency>
---------------------------------------------------------------

Regards,
Hawk

**Comment by xinglang** [comment]
Hi, 

Is there any issue during the maven build without it? I can build it with current pom file.

Best regards,
Xinglang

**Comment by hawkwang** [comment]
Really,

I got error messages as follows during "mvn package".

---------------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.0.2:compile (default-compile) on project jetstreamcore: Compilation failure: Compilation failure:
[ERROR] /home/hawkwang/Documents/tools/pulsarIO/jetstream/jetstreamcore/src/main/java/com/ebay/jetstream/servlet/JettyServer.java:[34,25] error: package org.mortbay.thread does not exist
[ERROR] 
[ERROR] /home/hawkwang/Documents/tools/pulsarIO/jetstream/jetstreamcore/src/main/java/com/ebay/jetstream/servlet/JettyServer.java:[35,25] error: package org.mortbay.thread does not exist
[ERROR] 
[ERROR] /home/hawkwang/Documents/tools/pulsarIO/jetstream/jetstreamcore/src/main/java/com/ebay/jetstream/servlet/JettyServer.java:[53,9] error: cannot access LifeCycle
[ERROR] 
[ERROR] class file for org.mortbay.component.LifeCycle not found
[ERROR] /home/hawkwang/Documents/tools/pulsarIO/jetstream/jetstreamcore/src/main/java/com/ebay/jetstream/servlet/JettyServer.java:[59,6] error: cannot find symbol
[ERROR] 
[ERROR] could not parse error message:   symbol:   method setHost(String)
[ERROR] location: class SimpleNIOConnector
[ERROR] /home/hawkwang/Documents/tools/pulsarIO/jetstream/jetstreamcore/src/main/java/com/ebay/jetstream/servlet/JettyServer.java:61: error: non-static method setPort(int) cannot be referenced from a static context
[ERROR] setPort(port);
[ERROR] ^
[ERROR] 
[ERROR] /home/hawkwang/Documents/tools/pulsarIO/jetstream/jetstreamcore/src/main/java/com/ebay/jetstream/servlet/JettyServer.java:[62,6] error: cannot find symbol
[ERROR] 
[ERROR] could not parse error message:   symbol:   method setUseDirectBuffers(boolean)
[ERROR] location: class SimpleNIOConnector
[ERROR] /home/hawkwang/Documents/tools/pulsarIO/jetstream/jetstreamcore/src/main/java/com/ebay/jetstream/servlet/JettyServer.java:70: error: cannot find symbol
[ERROR] configure(socket);

-----------------------------------------------------------------------------------------------------

And I tried to import projects into eclipse, and I found we miss somethong :-)
Pls have a check again!

Regards,
Hawk

**Comment by waater** [comment]
hey Hawk, 
if you look into [jetstream-parent](https://github.com/pulsarIO/jetstream/blob/master/Build/superpom/pom.xml#L576) there is already jetty dependency added. This pom is a parent of [jetstreamcore](https://github.com/pulsarIO/jetstream/blob/master/jetstreamcore/pom.xml#L3). 
What java version are you using?

**Comment by hawkwang** [comment]
Hey Waater,

I use "1.7.0_75", and I do not think it is java problem.
Yes, we have jetty, jetty-plus and jetty-naming as dependencies, however it seems
the following two statements in JettyServer.java need jetty-util .
import org.mortbay.thread.BoundedThreadPool;
import org.mortbay.thread.QueuedThreadPool;

And for me, I just do following steps to install jetstream library.
1) git clone https://github.com/pulsarIO/jetstream.git
2) cd jetstream
3) mvn install
And I got error message as I mentioned in my last comment :-)

Maybe I made wrong mistake here. 
Should I first "cd Build/superpom" and then use "mvn install"?

Regards,
Hawk
